packages/backend-common/src/service/lib/ServiceBuilderImpl.ts

```diff
@@ -45,9 +45,9 @@ export class ServiceBuilderImpl implements ServiceBuilder {
   // reloading
   private module: NodeModule;
 
-  constructor(module: NodeModule) {
+  constructor(moduleRef: NodeModule) {
     this.routers = [];
-    this.module = module;
```

Comment:
module here being language reserved symbol, causes weird failures when starting up the backend in dev mode